<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpUserInput">
     <property name="title">
      <string>Admin input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbLogin">
        <property name="text">
         <string>Login:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneLogin"/>
      </item>
      <item>
       <widget class="QLabel" name="lbPassword">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnePassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbError">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>LogIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LogIn()</slot>
 </slots>
</ui>
